lib/src/execute.rs

```diff
@@ -215,37 +221,20 @@ impl ExecuteCtx {
         );
 
         let resp = match receiver.await {
-            Ok(resp) => resp,
+            Ok(resp) => (resp, None),
             Err(_) => match guest_handle
                 .await
                 .expect("guest worker finished without panicking")
             {
-                Ok(_) => Response::new(Body::empty()),
+                Ok(_) => (Response::new(Body::empty()), None),
                 Err(ExecutionError::WasmTrap(_e)) => {
+                    eprintln!("There was an error handling the request {}", _e.to_string());
```

Comment:
We should use the following instead of `eprintln!` here
```suggestion
                    event!(Level::ERROR, "There was an error handling the request {}", _e.to_string());
```

Reply:
Done.